config/rbac/mcp_server_get_role.yaml

```
@@ -0,0 +1,21 @@
+      - deployments
+      - configmaps
+      - services
+      - routes
+      - pods
+      - namespaces
+      - projects
+      - mcpservers
+    verbs:
+      - get
+      - list
+      - watch
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)